### DISCUSSION TITLE
"nvidia is deprecated in favor of bestbuy"  when newegg and bestbuy are selected, not nvidia

### DISCUSSION THREAD

**Discussion by stg1232** [jef/streetmerchant#1279] [Help]
i edited the .env file to just be .env and still the problem persists. thought it might be that i still had the app for editing the file open, so i saved again and closed it, same issue. any idea what i need to do to get it to search bestbuy and newegg 

edit: the same thing pops up off a fresh install, nothing edited, no files changed

**Comment by SaltySeeds** [comment]
im having the same issue...

**Reply by SaltySeeds** [reply]
figured it out! delete the clone from your computer and re-clone it.. then repeat the process over again - this worked for me! seems like there was an issue with my prior .env file
